config/default_configs/default_config.yml

```diff
@@ -232,6 +232,9 @@ external_forcing:
 external_forcing_file:
   help: "External forcing file containing large-scale forcings, initial conditions, and boundary conditions [`nothing` (default), `path/to/file`]"
   value: ~
+cfsite_number:
+  help: "cfsite identifier for single column forcing from  `external_forcing_file`, specified as siteN. [`site23` (default), `siteN`]"
```

Comment:
Might be nice to link to Zhaoyi's paper so options are clear unless the list is somewhere else: https://doi.org/10.1029/2021MS002631